extra/import_data.py

```diff
@@ -142,8 +136,12 @@ def upload_to_mongodb(inheritance_analysis, accession_number, analyis_sources, a
     patient['annotation_sources'] = externalSources_dict  # add previous section to total patient info
 
     # extract information from the VUS/variant data and add to 'patient'
-    if vus_is_empty:
+    # Set default tot prevent empty field
+    patient['vus_is_empty'] = False
+    if vus_export == []:
```

Comment:
From the pep8 guideline: `For sequences, (strings, lists, tuples), use the fact that empty sequences are false`. 

Thus: 
```python
if not vus_export:
````